pulp_ansible/app/tasks/utils.py

```diff
@@ -216,3 +217,15 @@ def get_file_obj_from_tarball(tar, file_path, artifact_path, raise_exc=True):
         raise FileNotFoundError(f"{file_path} not found")
 
     return file_obj
+
+
+def get_last_synced_task_time(pk):
+    task = (
+        Task.objects.filter(reserved_resources_record__icontains=pk, name__icontains="sync")
+        .order_by("-pulp_last_updated")
+        .first()
+    )
+    if task:
+        return task.pulp_last_updated
```

Comment:
I would suggest using `pulp_created`. Multiple syncs can use the same remote and an older sync could run longer than a newer sync thus end up being the "lastest" sync when in fact it is just taking longer to complete.